documentation/modules/kafkaconnectors/proc-enabling-kafkaconnectors.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+
+[id='proc-enabling-kafkaconnectors-{context}']
+= Enabling `KafkaConnector` resources
+
+To enable `KafkaConnectors` for a Kafka Connect cluster, add the `strimzi.io/use-connector-resources` annotation to the `KafkaConnect` or `KafkaConnectS2I` custom resource. When enabled, connectors in the Kafka Connect cluster are created and managed using `KafkaConnectors`.
+
+.Prerequisites
+
+* A running Cluster Operator
+
+.Procedure
+
+. Edit a `KafkaConnect` or `KafkaConnectS2I` custom resource.
+
+. Add the `strimzi.io/use-connector-resources` annotation to the custom resource. For example:
```

Comment:
Why are these two separate bullets? They're one step in the procedure–the editing adds the annotation.